src/test/java/com/jcabi/github/mock/MkBranchTest.java

```diff
@@ -0,0 +1,117 @@
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the jcabi.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.github.mock;
+
+import com.jcabi.github.Coordinates;
+import com.jcabi.github.Repo;
+import java.io.IOException;
+import javax.json.Json;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test case for {@link MkBranch}.
+ *
+ * @author Chris Rebert (github@rebertia.com)
+ * @version $Id$
+ */
+public final class MkBranchTest {
+    /**
+     * Test user to own test repository.
+     */
+    private static final String REPO_USER = "jacqueline";
+    /**
+     * Test repository name.
+     */
+    private static final String REPO_NAME = "wonderful";
+
+    /**
+     * MkBranch should be able to fetch its name.
+     * @throws IOException If an I/O problem occurs
+     */
+    @Test
+    public void fetchesName() throws IOException {
+        final String name = "topic";
+        MatcherAssert.assertThat(
+            MkBranchTest.branches()
+                .create(name, "f8dfc75138a2b57859b65cfc45239978081b8de4")
+                .name(),
+            Matchers.equalTo(name)
+        );
+    }
+
+    /**
+     * MkBranch should be able to fetch its commit.
+     * @throws IOException If an I/O problem occurs
+     */
+    @Test
+    public void fetchesCommit() throws IOException {
+        final String sha = "ad1298cac285d601cd66b37ec8989836d7c6e651";
+        MatcherAssert.assertThat(
+            MkBranchTest.branches()
+                .create("feature-branch", sha).commit().sha(),
+            Matchers.equalTo(sha)
+        );
+    }
+
+    /**
+     * MkBranch should be able to fetch its repo.
```

Comment:
@cvrebert let's change to MkBranch can ... according to http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names